PKequal

luciano-casarini opened the pull request.

cmbant commented:
I think your branch conflicts with recent halofit changes (e.g. HMcode?),  Please merge with the latest version and resolve conflicts.

Please also revert changes to nonlinear.py and params.ini so that your new options do not change the default behaviour.

luciano-casarini commented:
I have seen there are some modifications of HMcode.
Now my fork is not more updated, and conflicts withe the updated CAMB.

Is it necessary that you provide me a new fork or not?
When I buit a new halofit_ppf.f90 how can i make a new pull request? Is it
possible from an old fork?

On Wed, Oct 19, 2016 at 3:16 PM, Antony Lewis <notifications@github.com>
wrote:

> I think your branch conflicts with recent halofit changes (e.g. HMcode?),
> Please merge with the latest version and resolve conflicts.
>
> Please also revert changes to nonlinear.py and params.ini so that your new
> options do not change the default behaviour.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cmbant/CAMB/pull/14#issuecomment-254879920>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/APNl-DZoTLpTORcHySRpHVKIaL6BDq7Dks5q1lB6gaJpZM4KbODO>
> .
>



-- 
Dr. Luciano Casarini

Visiting professor
Physics Dept. UFES
Vitoria, Brazil
Phone: +55 (27) 33352833
web: http://www.cosmo-ufes.org/luciano-casarini.html

cmbant commented:
I think you can use the same fork, you need to make a new pull request or update this one.

luciano-casarini commented:
the fork is not consistent with the original, a new halofit_ppf.90
compatible with
https://github.com/cmbant/CAMB
it will be not with
https://github.com/luciano-casarini/CAMB
is not it?

On Wed, Oct 19, 2016 at 4:22 PM, Antony Lewis <notifications@github.com>
wrote:

> I think you can use the same fork, you need to make a new pull request or
> update this one.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cmbant/CAMB/pull/14#issuecomment-254898434>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/APNl-FL1CDJ6t3caNn8F-tl51tt3OXX8ks5q1l_6gaJpZM4KbODO>
> .
>



-- 
Dr. Luciano Casarini

Visiting professor
Physics Dept. UFES
Vitoria, Brazil
Phone: +55 (27) 33352833
web: http://www.cosmo-ufes.org/luciano-casarini.html

cmbant commented:
You'd have to merge latest changes and fix conflicts.

luciano-casarini commented:
I do not know if I fixed the problem,
I changed the fork following your indications, and tested too.
I close and open again the pull request, but I guess the merging is not
automatic right?

On Wed, Oct 19, 2016 at 4:34 PM, Antony Lewis <notifications@github.com>
wrote:

> You'd have to merge latest changes and fix conflicts.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cmbant/CAMB/pull/14#issuecomment-254901696>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/APNl-ANogTrYnWYldr8vuPRSyUwIvFkvks5q1mKsgaJpZM4KbODO>
> .
>



-- 
Dr. Luciano Casarini

Visiting professor
Physics Dept. UFES
Vitoria, Brazil
Phone: +55 (27) 33352833
web: http://www.cosmo-ufes.org/luciano-casarini.html

cmbant commented:
You need to pull (and hence merge) the current master branch and fix conflicts.
https://git-scm.com/docs/git-pull

luciano-casarini commented:
I close and opened again the pull, but there is no merging.
I do not see any conflicts in  halofit_ppf.f90 and pycamb/camb/nonlinear.py  so I do not known why are indicated. I tested the code several times.

luciano-casarini commented:
I tried to add a file camb.f90 without change any on it, and it was included in the conflicts list. Maybe is there a problem of permissions ("Only those with write access to this repository can merge pull requests") ?

cmbant commented:
If you do "git pull cmbant master", where cmbant is a remote pointing to this git repository, you should get the merge on your local machine and be able to see and fix conflicts. (though given changes in whitespace etc, it may just be easier to start over with clean fork)
